docs/reference/persistence-configuration.txt

```diff
@@ -177,22 +188,52 @@ failing subsequent updates:
   band.with(collection: "artists") do |band_object|
     band_object.save!
   end
-  
+
   # This will not save - updates the collection "bands" which does not have
   # the Scuba band
   band.update_attribute(likes: 1000)
-  
+
   # This will update the document.
   band.with(collection: "artists") do |band_object|
     band_object.update_attribute(likes: 1000)
   end
 
-As of Mongoid 6.0, the ``with`` method must always be called with a block,
-and the temporary persistence context exists only for the duration of the block.
-This is because a new client is created under the covers with the options
-passed to ``with``. To ensure that this client is closed and its associated
-resources are freed, the scope when this client could be used must be
-well-defined.
+The ``with`` command can also be used without a block:
+
+.. code-block:: ruby
+
+  # The #with method on Model classes returns a criteria.
+  criteria = Band.with(database: "music-non-stop")
+  # => #<Mongoid::Criteria
+         selector: {}
+         options:  {}
+         class:    Band
+         embedded: false>
+
+  criteria.first
+  criteria.clear_persistance_context!
+
+  # The #with method on documents modifies the document object itself since
+  # calling #dup on it will change that document's _id. The #with method on
+  # documents will also return the same document.
+  band = Band.new
+  band.with(client: :tertiary)
+  band.save!
+  band.clear_persistance_context!
+
+  # The #with method on criteria returns a new criteria.
+  criteria = Band.all.with(collection: "artists")
+  band = criteria.first
+  criteria.clear_persistance_context!
```

Comment:
I suggest putting this line under an `ensure` block for proper cleanup. As written exceptions during result processing will leak clients.

Reply:
how's this?

Reply:
Should be `persistence` with an "e" as the more common spelling.